core/src/main/java/io/aiven/klaw/service/TopicOverviewService.java

```diff
@@ -133,8 +134,7 @@ private void enrichTopicInfoList(
     List<AclInfo> tmpAclPrefixed;
     List<AclInfo> tmpAcl;
     for (TopicInfo topicInfo : topicInfoList) {
-      aclsFromSOT.addAll(
-          getAclsFromSOT(topicInfo.getClusterId(), topicNameSearch, false, tenantId));
+      aclsFromSOT.addAll(getAclsFromSOT(topicInfo.getEnvId(), topicNameSearch, false, tenantId));
 
```

Comment:
more my education question was does SOT stand for?

Reply:
source of truth :)